.gitlab-ci.yml

```diff
@@ -0,0 +1,12 @@
+image: node:14
```

Comment:
Any reason why GitLab CI was added here?

Reply:
Yes, I designed/checked this PR in a private fork running on Gitlab prior to submitting it on GitHub... We can remove it now if you want. It is not harmful though.

Reply:
I have squashed the .gitlab-ci.yml commits into one

Reply:
Let's remove it 👍

Reply:
OK ,